nautobot/dcim/models/device_component_templates.py

```diff
@@ -195,12 +195,12 @@ class PowerOutletTemplate(ComponentTemplateModel):
     """
 
     type = models.CharField(max_length=50, choices=PowerOutletTypeChoices, blank=True)
-    power_port = models.ForeignKey(
+    power_port_template = models.ForeignKey(
         to="dcim.PowerPortTemplate",
         on_delete=models.SET_NULL,
         blank=True,
         null=True,
-        related_name="poweroutlet_templates",
+        related_name="power_outlet_templates",
```

Comment:
I am sure this makes sense, but why outlet and not ports?

Reply:
`related_name` is what the reverse-relation gets named from the related model back to this one. So here we have:

- (variable name given on `PowerOutletTemplate`) `PowerOutletTemplate.power_port_template`, a ForeignKey from a given `PowerOutletTemplate` to a single `PowerPortTemplate`
- (from the given `related_name`) `PowerPortTemplate.power_outlet_templates`, a [Manager](https://docs.djangoproject.com/en/3.2/topics/db/queries/#backwards-related-objects) back from a given `PowerPortTemplate` to the set of zero or more `PowerOutletTemplate` that reference it